### PR TITLE
Pal's Arcadia

### PR DESCRIPTION
**Pull request by irbahhana**

### CODE REVIEW

**Comment by ilmannafian04** [comment]
Looks pog to me!

File: src/routes/members.svelte [hunk 1]
```diff
@@ -10,6 +10,10 @@
       name: 'Wahyu Ananda Duli Tokan',
       username: 'wahyuadt',
     },
+    {
```
Comment (daystram) [inline src/routes/members.svelte:13]: Order by username

File: package-lock.json [hunk 2]
```diff
@@ -1,4687 +1,8 @@
 {
```
Comment (daystram) [inline package-lock.json:1]: Why removed?

File: src/routes/-/irbahhana.svelte [hunk 3]
```diff
@@ -0,0 +1,33 @@
+<script>
+</script>
+
```
Comment (daystram) [inline src/routes/-/irbahhana.svelte:3]: Add <Title />

File: src/routes/members.svelte (hunk 1 above)
Comment (ilmannafian04) [inline src/routes/members.svelte:13]: Please rebase `main` branch since #11 has already been merged

**Pull request merged**